core/Pos/Core/Constants.hs

```diff
@@ -80,14 +80,14 @@ isDevelopment = True
 isDevelopment = False
 #endif
 
--- | System start time embedded into binary.
-staticSysStart :: Maybe Timestamp
+-- | System start time embeded into binary.
+staticSysStart :: Timestamp
 staticSysStart
-    | isDevelopment = Nothing
-    | st > 0        = Just $ Timestamp $ sec st
+    | isDevelopment = Timestamp (sec 1)
```

Comment:
Why is it `sec 1`? i guess timestamp ~ unix time, so 1 sec is something not recent and it thu won't work.